pkg/scheduler/resource/relationship_check_task.go

```diff
@@ -162,6 +167,39 @@ func (in *RelationshipCheckTask) destroyResources(ctx context.Context) error {
 	return nil
 }
 
+// stopResources stops all resources that are in the progressing and stopping state.
+func (in *RelationshipCheckTask) stopResources(ctx context.Context) error {
+	resources, err := in.modelClient.Resources().Query().
+		Where(
+			func(s *sql.Selector) {
+				s.Where(sqljson.ValueEQ(
+					resource.FieldStatus,
+					summaryStatusProgressing,
```

Comment:
Is the summary status for stopping resources is `Progressing`?

Reply:
When it's waiting for dependencies, it's in `progressing` with the dependency message. When dependencies are resolved, it turns to `stopping` the job runs under the hood.